invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemPastLoans/state/actions.js

```diff
@@ -0,0 +1,32 @@
+import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
+import { loan as loanApi, serializeLoan } from '../../../../../../common/api';
+
+export const fetchPastLoans = (documentPid, itemPid) => {
+  return async dispatch => {
+    dispatch({
+      type: IS_LOADING,
+    });
+
+    await loanApi
+      .fetchLoans(
+        documentPid,
+        itemPid,
+        null,
+        null,
+        null,
+        'ITEM_RETURNED OR state:CANCELLED'
```

Comment:
Shouldn't it be also `state:ITEM_RETURNED`?

Reply:
nope, this is already in the function which creates the query. I just avoided here passing the array of few states.